charts/quickwit/templates/control-plane-deployment.yaml

```diff
@@ -19,6 +19,7 @@ spec:
         {{- toYaml . | nindent 8 }}
       {{- end }}
       labels:
+        {{- include "quickwit.labels" . | nindent 8 }}
```

Comment:
```suggestion
        {{- include "quickwit.podLabels" . | nindent 8 }}
```

Reply:
I'm not sure I understand what you mean here 😅

Reply:
Indeed, I could have explained myself better... Sorry.

Yeah, I'm not so sure we want all the labels to end up as pob labels but I'm probably over cautious. So nvm, it's doing the job for the user who asked for it.